Update README.md (#177)
open-edge-platform/edge-microvisor-toolkit: README.md

* [Edge Microvisor Toolkit Standalone Node (immutable)](https://edgesoftwarecatalog.intel.com/details/?microserviceType=recipe&microserviceNameForUrl=edge-microvisor-toolkit-standalone-node)
* [Edge Microvisor Toolkit Developer Node (mutable)](https://edgesoftwarecatalog.intel.com/details/?microserviceType=recipe&microserviceNameForUrl=edge--microvisor-toolkit-development-node)
* [Edge Microvisor Toolkit (immutable) - Provisioned from Edge Manageability Framework](https://github.com/open-edge-platform/edge-manageability-framework)

> * [Edge Microvisor Toolkit with real time extensions (immutable) - Provisioned from Edge Manageability Framework](https://github.com/open-edge-platform/edge-manageability-framework)

Reviewer: ```suggestion
* [Edge Microvisor Toolkit (immutable) - Provisioned in Edge Manageability Framework](https://github.com/open-edge-platform/edge-manageability-framework)
* [Edge Microvisor Toolkit with real time extensions (immutable) - Provisioned in Edge Manageability Framework](https://github.com/open-edge-platform/edge-manageability-framework)
```

Reviewer 2: Or maybe change "provisioned" to "available"? A more common word may be easier to read for people.

Reviewer: ```suggestion
* [Edge Microvisor Toolkit (immutable) - Available in Edge Manageability Framework](https://github.com/open-edge-platform/edge-manageability-framework)
* [Edge Microvisor Toolkit with real time extensions (immutable) - Available in Edge Manageability Framework](https://github.com/open-edge-platform/edge-manageability-framework)
```

**Edge Microvisor Toolkit Standalone Node**

> * [Build tools for bootable USB creator and how to install Edge Microvisor Toolkit Standalone Node](https://github.com/open-edge-platform/edge-microvisor-toolkit-standalone-node)

Reviewer: ```suggestion
* [Create a bootable USB installer for the Edge Microvisor Toolkit Standalone Node](https://github.com/open-edge-platform/edge-microvisor-toolkit-standalone-node)
```

> * [Build tools for bootable USB creator and how to install Edge Microvisor Toolkit Standalone Node](https://github.com/open-edge-platform/edge-microvisor-toolkit-standalone-node)

Reviewer 2: A recommendation:
the call to action should be consistent, for example: "Learn how to build tools and install EMTSN" or "Get the tools required to build USB creator and install EMFSN"

Reviewer: ```suggestion
* [Get the tools required to build USB creator and install Edge Microvisor Toolkit Standalone Node](https://github.com/open-edge-platform/edge-microvisor-toolkit-standalone-node)
```
